doc/api/sqlite.md

```diff
@@ -120,6 +122,28 @@ added: v22.5.0
 Closes the database connection. An exception is thrown if the database is not
 open. This method is a wrapper around [`sqlite3_close_v2()`][].
 
+### `database.loadExtension(path)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string} The path to the shared library to load.
+
+Loads a shared library into the database connection. The `allowLoadExtension` option must be
```

Comment:
`allowLoadExtension` is mentioned here. Should `enableLoadExtension()` be mentioned too?